plugin/client/client.py

```diff
@@ -15,19 +15,27 @@ class Spec:
     concurrency: int = field(default=DEFAULT_CONCURRENCY)
     queue_size: int = field(default=DEFAULT_QUEUE_SIZE)
     extract_utm: bool = field(default=False)
+    countries_summary_unit: str = field(default="month")
+    referrers_summary_unit: str = field(default="month")
+
+    def validate_summary_unit(self, unit: str, name: str):
+        if unit not in ["hour", "day", "week", "month"]:
+            raise Exception(f"{name} must be one of hour, day, week, month")
 
     def validate(self):
         if self.api_token is None:
             raise Exception("api_token must be provided")
         if self.group_id is None:
             raise Exception("group_id must be provided")
+        self.validate_summary_unit(self.countries_summary_unit, "countries_summary_unit")
+        self.validate_summary_unit(self.referrers_summary_unit, "referrers_summary_unit")
 
 
 class Client(ClientABC):
     def __init__(self, spec: Spec) -> None:
         self._spec = spec
         self._client = BitlyClient(
-            spec.api_token, spec.group_id, spec.base_url, spec.extract_utm
+            spec.api_token, spec.group_id, spec.base_url, spec.extract_utm, spec.countries_summary_unit, spec.referrers_summary_unit
```

Comment:
Was this formatted by `black`? I would have assumed it'd break the line

Reply:
forgot to run it :)